<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>studio25 - bespoke software</author>
 <class>BatchRename</class>
 <widget class="QWidget" name="BatchRename">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>664</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>960</width>
    <height>664</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>960</width>
    <height>664</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Calibri</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Batch Renamer</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255)</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>941</width>
     <height>501</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="previewGroup">
    <item row="1" column="0">
     <widget class="QLabel" name="previewTargetsLabel">
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Targets:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QListView" name="previewOutput">
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="toolTip">
       <string>Preview of output file names</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QListView" name="previewTargets">
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="toolTip">
       <string>Target files found in the selected directory</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="previewOutputLabel">
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Preview:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>550</y>
     <width>941</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="operationGroup">
    <item>
     <widget class="QLabel" name="operationLabel">
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Rename:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="operationInput">
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>C:\some\file.jpg</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="operationArrow">
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>--&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="operationOutput">
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>name001.jpg</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="operationSeperator">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>580</y>
     <width>941</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="progressGroup">
    <item>
     <widget class="QProgressBar" name="progressBar">
      <property name="palette">
       <palette>
        <active>
         <colorrole role="Button">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Window">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Highlight">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>110</red>
            <green>127</green>
            <blue>210</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="Button">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Window">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Highlight">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>110</red>
            <green>127</green>
            <blue>210</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="Button">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Window">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Highlight">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>0</green>
            <blue>128</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="textVisible">
       <bool>true</bool>
      </property>
      <property name="invertedAppearance">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="Line" name="progressSeperator">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="progressLabel">
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>6/27</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="RenameButton">
   <property name="geometry">
    <rect>
     <x>847</x>
     <y>620</y>
     <width>101</width>
     <height>30</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Perform the batch rename operation</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #afcfe2;</string>
   </property>
   <property name="text">
    <string>Rename!</string>
   </property>
  </widget>
  <widget class="Line" name="BottomSpacer">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>620</y>
     <width>831</width>
     <height>31</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>461</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="directoryGroup">
    <item>
     <widget class="QLabel" name="directoryLabel">
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Directory:</string>
      </property>
      <property name="buddy">
       <cstring>directoryInput</cstring>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="directoryInput">
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="toolTip">
       <string>Select the directory containing the image files to be renamed</string>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #aabaf2; color: #FFFFFF;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="directoryBrowse">
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #afcfe2;</string>
      </property>
      <property name="text">
       <string>Browse ...</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>688</x>
     <y>10</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="basenameGroup">
    <item>
     <widget class="QLabel" name="basenameLabel">
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Basename:</string>
      </property>
      <property name="buddy">
       <cstring>basenameInput</cstring>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="basenameInput">
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="toolTip">
       <string>Choose the base name for the renamed files</string>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #aabaf2; color: #FFFFFF;</string>
      </property>
      <property name="text">
       <string>base</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>directoryInput</sender>
   <signal>editingFinished()</signal>
   <receiver>BatchRename</receiver>
   <slot>changeDirectory()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>265</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>274</x>
     <y>15</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>directoryBrowse</sender>
   <signal>released()</signal>
   <receiver>BatchRename</receiver>
   <slot>browseDirectory()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>469</x>
     <y>29</y>
    </hint>
    <hint type="destinationlabel">
     <x>474</x>
     <y>35</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>basenameInput</sender>
   <signal>editingFinished()</signal>
   <receiver>BatchRename</receiver>
   <slot>changeBasename()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>781</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>793</x>
     <y>26</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>RenameButton</sender>
   <signal>released()</signal>
   <receiver>BatchRename</receiver>
   <slot>rename()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>847</x>
     <y>620</y>
    </hint>
    <hint type="destinationlabel">
     <x>855</x>
     <y>620</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>perform()</signal>
  <slot>rename()</slot>
  <slot>changeDirectory()</slot>
  <slot>changeBasename()</slot>
  <slot>browseDirectory()</slot>
 </slots>
</ui>
